manifest.toml

```diff
@@ -58,12 +59,13 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://downloads.wordpress.org/release/wordpress-6.6.2.zip"
-        sha256 = "f37fcc6b6875ad6e6162125b3bbe6a91d65b0aebf6c4fa3b6af3582b38d1418e"
+        url = "https://downloads.wordpress.org/release/wordpress-6.7.zip"
+        sha256 = "78b07b760edb0d30eb3233fdb6c68cf6a2ba4a2c20309f3cfb5d00432edcff92"
 
     [resources.system_user]
 
     [resources.install_dir]
+    group = "www-data:r-x"
```

Comment:
We want WordPress to automatically upgrade for minor versions, I think `rwx` is necessary here